includes/ConsoleTasks/RunJobQueueTask.php

```diff
@@ -106,13 +106,19 @@ public function execute()
                 $database->rollBack();
                 try {
                     $database->beginTransaction();
-
+                    
                     /** @var JobQueue $job */
                     $job = JobQueue::getById($job->getId(), $database);
                     $job->setDatabase($database);
-                    $job->setStatus(JobQueue::STATUS_FAILED);
-                    $job->setError($ex->getMessage());
-                    $job->setAcknowledged(0);
+                    if (str_contains($ex->getMessage, 'mwoauth-invalid-authorization')) {
+                        $job->setAcknowledged(1);
+                        $job->setError('Receieved error with authorization headers. Will retry');
+                        $job->setStatus(JobQueue::STATUS_READY);
+                    } else {
+                        $job->setStatus(JobQueue::STATUS_FAILED);
+                        $job->setError($ex->getMessage());
+                        $job->setAcknowledged(0);
+                    }
                     $job->save();
 
                     Logger::backgroundJobIssue($this->getDatabase(), $job);
```

Comment:
I think it'd be good to generate a requeue log entry as well to make it more clear what's going on.